From: James Clarke <jrtc27@jrtc27.com>
Date: Fri, 26 Apr 2019 23:32:46 +0100
Subject: ia64: Don't clear register_stack_start

r59829 stopped clearing stack_start and enabled the code for
!FIBER_USE_NATIVE, but we need to do the same for register_stack_start
on ia64, otherwise we end up with NULL in cont_save_machine_stack.

Closes: https://github.com/ruby/ruby/pull/2155
---
 cont.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/cont.c b/cont.c
index 7b6864a..8ad8207 100644
--- a/cont.c
+++ b/cont.c
@@ -543,7 +543,6 @@ cont_save_thread(rb_context_t *cont, rb_thread_t *th)
     sec->machine.stack_end = NULL;
 
 #ifdef __ia64
-    sec->machine.register_stack_start = NULL;
     sec->machine.register_stack_end = NULL;
 #endif
 }
